DipAdvisor/app/screens/SingleLocationScreen.js

```diff
@@ -90,23 +129,42 @@ function SingleLocationScreen({ route, navigation }) {
       </View>
       <View style={styles.info}>
         <Text style={styles.title}>{location.location_name}</Text>
-        <Text style={styles.description}>{location.description}</Text>
-        <View style={styles.propertiesList}>
-          <FlatList
-            data={[
-              { name: "Depth", value: location.depth ? location.depth : "N/A" },
-              { name: "Public", value: location.public ? "Yes" : "No" },
-              {
-                name: "Water Temperature",
-                value: location.water_temp ? location.water_temp : "N/A",
-              },
-            ]}
-            keyExtractor={(item, index) => index.toString()}
-            renderItem={renderLocationProperty}
-          />
-        </View>
+        <FlatList
+          data={[
+            {
+              name: "Depth",
+              value: location.depth ? location.depth : "N/A",
+            },
+            { name: "Public", value: location.public ? "Yes" : "No" },
+            {
+              name: "Water Temperature",
+              value: location.water_temp ? location.water_temp : "N/A",
+            },
+          ]}
+          keyExtractor={(item, index) => index.toString()}
+          renderItem={renderLocationProperty}
+        />
+        <ScrollView>
+          <Text style={styles.description}>{location.description}</Text>
+          <Button title="add pic" onPress={() => pickImage(setImage)} />
+          {image && (
+            <View>
+              <Image source={{ uri: image }} style={{ width: 12, height: 9 }} />
+              <Button
+                title="upload image"
+                onPress={() => {
+                  uploadImage(image, setImageUrl);
+                }}
+              />
+              <Button
+                title="add image to location"
+                onPress={() => handleAddPhotoToLocation()}
+              />
+            </View>
+          )}
+        </ScrollView>
```

Comment:
This scroll view doesn't work and I need help making the buttons appear/disappear.